pandas/tests/io/test_clipboard.py

```diff
@@ -238,10 +238,11 @@ def test_read_clipboard_infer_excel(self, request, mock_clipboard):
         tm.assert_frame_equal(res, exp)
 
     def test_invalid_encoding(self, df):
+        msg = "invalid coding, encoding must be ascii"
         # test case for testing invalid encoding
-        with pytest.raises(ValueError):
+        with pytest.raises(ValueError, match=msg):
```

Comment:
looking at the CI the message being thrown is different

Reply:
I thought that the message didn't exist and wrote one, I'll fix them so they match with what's supposed to say. Where can I find the exact message?

Reply:
run the code that throws the error